yrs/src/block.rs

```diff
@@ -148,8 +163,7 @@ impl Block {
     pub fn last_id(&self) -> ID {
         match self {
             Block::Item(item) => item.last_id(),
-            Block::Skip(skip) => ID::new(skip.id.client, skip.id.clock + skip.len),
```

Comment:
Since `Block::Skip` doesn't exists in block store (only used in context of update), I removed this case from default `Block` definition.